shell/common/rasterizer.cc

```diff
@@ -297,21 +298,16 @@ RasterStatus Rasterizer::DoDraw(
     }
     const auto frame_lag =
         (latest_frame_target_time - frame_target_time).ToMillisecondsF();
-    const int vsync_transitions_missed = round(frame_lag / frame_budget_millis);
-    fml::tracing::TraceEventAsyncComplete(
-        "flutter",                    // category
-        "SceneDisplayLag",            // name
-        raster_finish_time,           // begin_time
-        latest_frame_target_time,     // end_time
-        "frame_target_time",          // arg_key_1
-        frame_target_time,            // arg_val_1
-        "current_frame_target_time",  // arg_key_2
-        latest_frame_target_time,     // arg_val_2
-        "vsync_transitions_missed",   // arg_key_3
-        vsync_transitions_missed      // arg_val_3
-    );
+    vsync_transitions_missed =
+        std::to_string(round(frame_lag / frame_budget_millis));
```

Comment:
Should we use `ceiling` instead of `round` here? Say if a frame has a lag of `0.1` frame, it seems to still cause 1 vsync transition to be missed.

Reply:
I can see both sides to the argument here. If there is `0.1` frame, there might be too much variance to do much to avoid this. It might be "too much" penalty on the app developer to charge a full frame worth of lag. That was the intent in using `round`.

One other alternative is to keep this as a `float` instead of `int`, we can retain say `.2f` precision. WDYT?

Reply:
Hmm... maybe I misunderstood the definition of `vsync_transitions_missed` as it was `round` before (and I'm Ok to just use `round` in this PR as this PR is just fixing the async events).

To better understand the definition of `vsync_transitions_missed`, I went back and checked the examples in https://github.com/flutter/engine/pull/17384#discussion_r401893796. That example says `frame_lag = 17 (ms), missed = 2`. Since a frame is roughly 16ms, the 17 (ms) frame_lag is a lag of `1.0625` frame. Then only a `ceil` would make it `2` instead of `1`. It seems that we intended to give "too much" penalty even for a little lag above a single frame time?